tray_id:
  part_4 : "tray_1_partition_4"
  part_5 : "tray_2_partition_6"
  part_6 : "tray_1_partition_3"
  part_7 : "tray_1_partition_2"
  part_8 : "tray_2_partition_1"
  part_9 : "tray_2_partition_4"
  part_10: "tray_2_partition_7"
  part_11: "tray_1_partition_1"
  part_12: "tray_2_partition_3"
  part_13: "tray_1_partition_5"
  part_14: "tray_2_partition_2"
  part_15: "tray_2_partition_5"
  part_16: "tray_2_partition_8"
